WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -198,6 +198,33 @@ function(ex) {
     }
 };
 
+ZmTaskListView.prototype._getLabelFieldList =
+function () {
+	return [
+		ZmItem.F_SELECTION,
+		ZmItem.F_NAME,
+		ZmItem.F_PRIORITY
+	];
+};
+
+ZmTaskListView.prototype._getLabelForField =
+function(item, field) {
+	switch (field) {
+		case ZmItem.F_NAME:
+			return item.name;
+		case ZmItem.F_PRIORITY:
+			if (item.isHighPriority) {
+				return ZmMsg.priorityHigh;
+			} else if (item.isLowPriority) {
+				return ZmMsg.priorityLow;
+			} else {
+				return ZmMsg.priorityNormal;
+			}
```

Comment:
```suggestion
			if (item.isHighPriority) {
				return ZmMsg.priorityHigh;
			}
			if (item.isLowPriority) {
				return ZmMsg.priorityLow;
			}
			return ZmMsg.priorityNormal;
```

Reply:
it's either or condition so current version looks better

Reply:
Done.